docs/articles_en/documentation/openvino-extensibility.rst

```diff
@@ -124,9 +124,9 @@ The ``Identity`` is a custom operation class defined in :doc:`Custom Operation G
          :language: cpp
          :fragment: [add_frontend_extension]
 
-When Python API is used, there is no way to implement a custom OpenVINO operation. Even if custom OpenVINO operation is implemented in C++ and loaded into the runtime by a shared library, there is still no way to add a frontend mapping extension that refers to this custom operation. In this case, use C++ shared library approach to implement both operations semantics and framework mapping.
+The Python API now supports implementation of custom OpenVINO operations, enabling direct integration within Python code. Still python custom operations won't be loaded into the runtime, so use the C++ shared library approach to implement both operation semantics and framework mapping that refers to this custom operation.
```

Comment:
This is not a release notes, I suppose. So `now` is redundant.

Reply:
The meaning of the whole sentence doesn't look correct: "Still python custom operations won't be loaded into the runtime, so use the C++ shared library approach to implement both operation semantics and framework mapping that refers to this custom operation.". By definition of Python custom operation you already have "loaded" it in runtime - just insert it into the graph and it will work. Am I missing something?